### PR TITLE
Prepare a final CM OS package

### PR DESCRIPTION
**Pull request by berezovskyi**
See https://issues.oasis-open.org/browse/TCADMIN-3996 for OS publication progress

See https://github.com/oasis-tcs/tab-respec/pull/51 for the list of changes to the templates

### CODE REVIEW

**Review by jamsden — APPROVED** [review]
Looks good to me.

**Comment by berezovskyi** [comment]
Thanks Jim! Will merge once the ReSpec PR is merged and v2.1.25 is released.

**Pull request merged**
